examples/test_langchain_llm_output.metta

```diff
@@ -0,0 +1,102 @@
+!(import! &self motto)
+(= (doc joke_struct)
+  (Doc
+    (type "object")
+    (description "joke description")
+    (properties
+       (item
+           (anyOf
+                   is_array
+                   (
+                       (description "Joke to tell user.")
+                        (properties
+                          (setup
+                            (description "The setup of the joke")
+                            (type "string")
+                          )
+                           (punchline
+                            (description "The punchline to the joke")
+                            (type "string")
+                          )
+                            (rating
+                                (type "integer")
+                                (description "How funny the joke is, from 1 to 10")
+                                (default  None)
+                            )
+                        )
+                        (type "object")
+                        (required is_array setup punchline)
+
+                     )
+                     (
+                        (description "Joke that user said.")
+                        (properties
+
+                           (punchline
+                            (description "The punchline to the joke")
+                            (type "string")
+                          )
+
+                            (rating
+                                (type "integer")
+                                (description "How funny the joke is, from 1 to 10")
+                                (default  None)
+                            )
+                        )
+                        (type "object")
+                        (required is_array punchline)
+                     )
+
+             )
+          )
+      )
+     (required is_array item)
+  )
+)
+
+
+!(import! &self motto)
+(= (doc multiply)
+  (Doc
+    (description "Multiplies a and b")
+    (properties
+      (a
+        (title 'A')
+        (type 'integer')
+      )
+       (b
+        (title 'B')
+        (type 'integer')
+      )
+    )
+     (type 'object')
+     (required is_array a b)
+  )
+)
+
+(= (doc multiply_struct)
```

Comment:
What's the point of having structured output in the same format as input to the tool? Will the structured output be used if the tool is used? It would be nice to (also) have an example of structured output converted to metta expressions.